scripts/jwst/build_parent_sample.py

```diff
@@ -1,404 +1,352 @@
-    file = wget.download(base_url)
-
-    # Read the content of the file
-    with open(file, "r") as f:
-        file_content = f.read()
-
-    # Parse the HTML content
-    soup = BeautifulSoup(file_content, "html.parser")
-
-    jwstfiles = []
-    for temp in soup.find_all("a"):
-        if (field_identifier in temp["href"]) and ("_sci" in temp["href"]):
-            jwstfiles.append(temp["href"])
-
-    # Print and download files
-    for url in jwstfiles:
-        # Extract the filename from the URL
-        filename = url.split("/")[-1]
-
-        # Determine the filter from the filename
-        temp = filename.split("_")[0].split("-")
-        if temp[-1] != "clear":
-            filter_name = temp[-1]
-        else:
-            filter_name = temp[-2]
-
-        # Construct the full local filepath
-        full_local_path = os.path.join(output_directory, filename)
-        # Download the file to the specified output directory
-        if (filter_name in filter_list) and (not os.path.isfile(full_local_path)):
-            print(f"... {full_local_path} does not exist - downloading from {url}...")
-            wget.download(url)
-
-    photoz_url = None
-    phot_url = None
-
-    relevant_hrefs = [a.get('href') for a in soup.find_all('a', href=lambda href: href and field_identifier in href)]
-    for href in relevant_hrefs:
-        if "photoz" in href:
-            photoz_url = href
-        if "phot_apcorr.fits" in href:
-            phot_url = href
-        
-        if photoz_url and phot_url:
-            break  # Exit the loop if both URLs are found
-
-    if not photoz_url:
-        raise ValueError(f"No photoz file found for {field_identifier}")
-    if not phot_url:
-        raise ValueError(f"No photometry file found for {field_identifier}")
-
-    # download the photoz file
-    filename = photoz_url.split("/")[-1]
-    full_local_path = os.path.join(output_directory, filename)
-    if not os.path.isfile(full_local_path):
-        print(f"... Downloading to {full_local_path}...")
-        file = wget.download(photoz_url)
-        # unzip the file
-        tar = tarfile.open(file)
-        tar.extractall()
-        tar.close()
-
-    # download photometric catalog
-    filename = phot_url.split("/")[-1]
-    if not os.path.isfile(full_local_path):
-        wget.download(phot_url)
-    phot_table = Table.read(filename)
-
-    # read it in as a table
-    fnames = os.listdir(".")
-    for fname in fnames:
-        if "eazypy.zout" in fname:
-            photz_table = Table.read(fname)
-
-    joint_table = join(
-        phot_table,
-        photz_table,
+_cutout_size = 96
+_mosaics = [
+    # Primer fields
+    "primer-cosmos-east-grizli-v7.0", "primer-cosmos-west-grizli-v7.0",
+    "primer-uds-north-grizli-v7.2", "primer-uds-south-grizli-v7.2",
+    # CEERS fields
+    "ceers-full-grizli-v7.4",
+    # NGDEEP fields
+    "ngdeep-grizli-v7.2",
+    # GOODS fields
+    "gds-grizli-v7.2", "gdn-grizli-v7.3",
```

Comment:
Why aren't we processing / formatting more from the [DJA index](https://s3.amazonaws.com/grizli-v2/JwstMosaics/v7/index.html)?

Reply:
because these are the fields we documented in the paper

Reply:
Agreed. I think for a v2, it would be good to add support for them. :)